### ISSUE TITLE
Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by Khyati007** [khyatit13/Coding-practice#224]
Source: `<span class="c-coordinateJobPanel__gridFieldText ng-star-inserted" title="Ready to Work">Ready to Work</span>`
Element Location: .p-highlight.p-selectable-row > .ui-resizable-column:nth-child(6) > .c-coordinateJobPanel__gridFieldText[title="Ready to Work"]
failureSummary: Fix any of the following:
  Element has insufficient color contrast of 4.49 (foreground color: #6a6a6a, background color: #d4eefe, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1Tags: [Ljava.lang.String;@3546d80f
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
